src/Entity/Comment.php

```diff
@@ -105,7 +105,7 @@ public function setContent(string $content): void
         $this->content = $content;
     }
 
-    public function getPublishedAt(): \DateTime
+    public function getPublishedAt(): ?\DateTime
```

Comment:
not needed, as the field is not nullable in the DB and the constructor initializes it